src/components/Todo/Todo.tsx

```diff
@@ -11,24 +11,43 @@ export const Todo: React.FC<TodoProps> = ({
   descr,
   deleteTodo,
   handleEditVisibleModal,
+  avaliableOptions,
+  selectedOption,
+  handleOption,
+  classes,
 }) => {
   return (
     <div className="todo">
       <div className="todo-text">
         <h2 className="todo-text__title">{title}</h2>
         <h3 className="todo-text__descr">{descr}</h3>
       </div>
-      <div className="todo-buttons">
-        <RoundButton
-          btnText="X"
-          btnClass="round-btn round-btn--red"
-          onClickBtn={() => deleteTodo(id)}
-        />
-        <RoundButton
-          btnText="E"
-          btnClass="round-btn round-btn--yellow"
-          onClickBtn={() => handleEditVisibleModal(id)}
-        />
+      <div className="todo-right-part">
+        <select
+          name="todo-select"
+          id="todo-select"
+          onChange={handleOption}
+          className={classes}
+          value={selectedOption}
+        >
+          {avaliableOptions.map((option) => (
+            <option key={option} value={option}>
+              {option}
+            </option>
+          ))}
+        </select>
+        <div className="todo-buttons">
+          <RoundButton
+            btnText="X"
+            btnClass="round-btn round-btn--red"
+            onClickBtn={() => deleteTodo(id)}
+          />
+          <RoundButton
+            btnText="E"
+            btnClass="round-btn round-btn--yellow"
+            onClickBtn={() => handleEditVisibleModal(id)}
+          />
```

Comment:
There is logic from another ticket. Wait until it'll be merged or delete it

Reply:
deleted